packages/storybook/src/nimble/number-field/number-field-matrix.stories.ts

```diff
@@ -73,34 +76,69 @@ const component = (
         ?hide-step="${() => hideStep}"
         ?readonly="${() => readonly}"
         ?disabled="${() => disabled}"
+        ?appearance-readonly="${() => appearanceReadOnly}"
         error-text="${() => errorText}"
         ?error-visible="${() => errorVisible}"
         ?required-visible="${() => requiredVisible}"
     >
         ${() => errorName} ${() => appearanceName} ${() => valueName}
-        ${() => hideStepName} ${() => disabledName} ${() => readOnlyName}
+        ${() => hideStepName} ${() => disabledReadOnlyName}
         ${() => requiredVisibleName}
     </${numberFieldTag}>
 `;
 
-export const themeMatrix: StoryFn = createMatrixThemeStory(
+const [
+    lightThemeWhiteBackground,
+    colorThemeDarkGreenBackground,
+    darkThemeBlackBackground,
+    ...remaining
+] = backgroundStates;
+
+if (remaining.length > 0) {
+    throw new Error('New backgrounds need to be supported');
+}
+
+export const lightTheme: StoryFn = createFixedThemeStory(
+    createMatrix(component, [
+        requiredVisibleStates,
+        disabledReadOnlyStates,
```

Comment:
nit / pref, could we move up the appearance states so those controls are grouped together?
```
disabledReadOnlyStates,
appearanceStates,
requiredVisibleStates,
hideStepStates,
valueStates,
errorStates
```

Guess a concept from that could be that larger dimensions should be first so that you have bigger groupings spread out (like 8 different categories for disabled states) and then binary dimensions last which keeps the smaller sets of variations closer together.